spec/index.html

```diff
@@ -225,30 +301,29 @@
           attribute boolean? expanded;
           attribute boolean? disabled;
           attribute InvalidType? invalid;
-          attribute boolean hasPopUp;
+          attribute boolean? hasPopUp;
+          attribute boolean? pressed;
 
           // Control values.
-          attribute DOMString? value;
+          attribute DOMString? valueText;
           attribute DOMString? placeholder;
-          attribute double? rangeValue;
-          attribute double? rangeMin;
-          attribute double? rangeMax;
-          attribute double? rangeStep;
+          attribute double? valueNow;
+          attribute double? valueMin;
+          attribute double? valueMax;
 
           // Live regions.
-          attribute boolean? liveAtomic;
-          attribute boolean? liveBusy;
-          attribute LiveType? liveStatus;
-          attribute LiveRelevantType? liveRelevant;
+          attribute boolean? atomic;
+          attribute boolean? busy;
+          attribute LiveType? live;
+          attribute LiveRelevantType? relevant;
```

Comment:
I would prefer ARIA live regions be left out entirely (ultimately an AOM `announcementNotification` should negate the need) but if these are left in, let's keep the naming prefix "live*" so it's obvious these are all related.